src/main/java/com/cloudogu/smeagol/wiki/infrastructure/PageResourceAssembler.java

```diff
@@ -64,6 +65,10 @@ private Link selfLink(Page page) {
         return baseLink(page).withSelfRel();
     }
 
+    private Link deleteLink(Page page) {
+        return baseLink(page).withRel("delete");
+    }
```

Comment:
I could not find a test for the delete link.

Reply:
Is that ok? a2b3419b3cf4d5e1ad08bfb46e275e2ec14f5a6d

Reply:
:+1: